tc-vars.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -ex
+
+HOME_CLEAN=$(/usr/bin/realpath "${HOME}")
+
+PATH=${HOME_CLEAN}/bin/:$PATH
+export PATH
+
+LD_LIBRARY_PATH=${HOME_CLEAN}/DeepSpeech/CUDA/lib64/:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH
+
+BAZEL_URL=https://github.com/bazelbuild/bazel/releases/download/0.4.2/bazel-0.4.2-installer-linux-x86_64.sh
+BAZEL_SHA256=b76b62a8c0eead1fc2215699382f1608c7bb98529fc48c5e9ef3dfa1b8b7585e
+
+CUDA_URL=https://developer.nvidia.com/compute/cuda/8.0/prod/local_installers/cuda_8.0.44_linux-run
```

Comment:
They are used by `tc-setup.sh`, which sources `tc-vars.sh`. What kind of groupping exactly would you like?